secant/Features/App/AppStore.swift

```diff
@@ -454,7 +454,7 @@ extension AppStore {
     static var placeholder: AppStore {
         AppStore(
             initialState: .placeholder,
-            reducer: AppReducer()
+            reducer: AppReducer()._printChanges()
```

Comment:
I think this might have been accidentally committed

Reply:
that is by purpose @pacu, it’s a replacement for the .debug() on a reducer and it prints all changes of the state for the debugging… it’s handled on the TCA side that it won’t print anything for release build